Cargo.toml

```diff
@@ -49,7 +49,7 @@ lazy_static = "1"
 rusty-fork = "0.3.0"
 
 [target.'cfg(windows)'.dependencies]
-windows-sys = { version = "0.48.0", features = ["Win32_Globalization"] }
+windows-targets = { version = "0.52.0" }
```

Comment:
Note that while this still needs something from the `windows-*` ecosystem, they've made `windows-targets` pretty stable and it hasn't had a breaking change in over a year.